lib/net/pool.js

```diff
@@ -3318,6 +3364,10 @@ class Pool extends EventEmitter {
     if (!this.opened)
       return;
 
+    // TODO: this saves compute by exiting early
+    // Now there are two different settings. Without
+    // an argument to `addOutbound`, there is no way to
+    // exit early like this.
```

Comment:
I think this problem could be solved with something like:

```js
if (this.peers.outbound >= this.options.maxOutbound
    && this.peers.outboundBrontide >= this.options.maxOutboundBrontide)
  return;
```

Then it would involve determining which type (brontide or plaintext) could be added and getting that type from `getHost`